binary/cli/cli_test.go

```diff
@@ -168,6 +168,13 @@ func TestValidateFlags(t *testing.T) {
 			},
 			wantErr: cmpopts.AnyError,
 		},
+		{
+			desc: "Image Platform without Remote Image",
+			flags: &cli.Flags{
+				ImagePlatform: "linux/amd64",
```

Comment:
Can you also add a test case where both are set and this doesn't return an error?